src/Microsoft.AspNetCore.Hosting.Abstractions/WebHostDefaults.cs

```diff
@@ -6,6 +6,7 @@ namespace Microsoft.AspNetCore.Hosting
     public static class WebHostDefaults
     {
         public static readonly string ApplicationKey = "application";
+        public static readonly string EnvironmentApplicationKey = "environmentApplication";
```

Comment:
Well a more descriptive one would be `ApplicationEnvironmentApplicationName` but maybe that's a little too verbose? I'm just prepending what we use for Application currently.